File: euler/1/1.f
\ /usr/bin/env pforth
: findsumofmultiples ( to -- sum )
 0 swap
 1 do
  I 3 mod
  I 5 mod
   *
  0=
  if I + then
 loop
;
 
1000 findsumofmultiples

cr 
." Result: " .
cr

File: Euler/1/1.lua
#!/usr/bin/env lua

function solution(n)
 local sum = 0
 for i=3,n-1 do
  if i%3==0 or i%5==0 then
   sum = sum + i
  end
 end
 return sum
end

print( solution(arg[1] or 1000) )
